frontend/rust-lib/flowy-sqlite/src/schema.rs

```diff
@@ -101,6 +101,7 @@ diesel::table! {
         created_at -> BigInt,
         database_storage_id -> Text,
         icon -> Text,
+        member_count -> BigInt,
```

Comment:
Do we need to handle up.sql and down.sql manually?

Reply:
yeah, I guess you are right. I missed that

Reply:
added the `up.sql` and `down.sql` to `frontend/rust-lib/flowy-sqlite/migrations`